Mathlib/Data/List/FinRange.lean

```diff
@@ -25,48 +25,46 @@ namespace List
 variable {α : Type u}
 
 @[simp]
-theorem map_coe_fin_range (n : ℕ) : (finRange n).map coe = List.range n :=
-  by
-  simp_rw [fin_range, map_pmap, Fin.val_mk, pmap_eq_map]
+theorem map_coe_finRange (n : ℕ) : ((finRange n) : List (Fin n)).map (Fin.val) = List.range n := by
+  simp_rw [finRange, map_pmap, Fin.val_mk, pmap_eq_map]
   exact List.map_id _
-#align list.map_coe_fin_range List.map_coe_fin_range
+#align list.map_coe_fin_range List.map_coe_finRange
 
-theorem fin_range_succ_eq_map (n : ℕ) : finRange n.succ = 0 :: (finRange n).map Fin.succ :=
-  by
+theorem finRange_succ_eq_map (n : ℕ) : finRange n.succ = 0 :: (finRange n).map Fin.succ := by
   apply map_injective_iff.mpr Fin.val_injective
-  rw [map_cons, map_coe_fin_range, range_succ_eq_map, Fin.val_zero, ← map_coe_fin_range, map_map,
-    map_map, Function.comp, Function.comp]
-  congr 2 with x
-  exact (Fin.val_succ _).symm
-#align list.fin_range_succ_eq_map List.fin_range_succ_eq_map
-
-@[simp]
+  rw [map_cons, map_coe_finRange, range_succ_eq_map, Fin.val_zero, ← map_coe_finRange, map_map,
+    map_map]
+  simp only [Function.comp, Fin.val_succ]
+#align list.fin_range_succ_eq_map List.finRange_succ_eq_map
+
+-- Porting note TODO: we have List.finRange_map_get in Data.List.Range, which seems to be exactly
+-- this but currently not a simp lemma. Should we remove map_nth_le, make finRange_map_get
+-- a simp lemma and align map_nth_le with finRange_map_get?
```

Comment:
Yes, `nthLe` is deprecated, so we always align with `get`-lemmas instead.
If you add the `#align`-statement in the other file, please leave a porting note here with a pointer.

Reply:
done, making it at `simp` lemma like it used to be